src/deployment/cd.md

```diff
@@ -193,6 +193,111 @@ secrets in pull requests that you accept and merge.
          * `cd android` or `cd ios`
          * `bundle exec fastlane [name of the lane]`
 
+## Xcode Cloud
+
+[Xcode Cloud][] is a continuous integration and delivery service for building,
+testing, and distributing apps and frameworks for Apple platforms.
+
+### Requirements
+
+* Xcode 13.4.1 or higher.
+* Be enrolled in the [Apple Developer Program][].
+
+### Custom build script
+
+Xcode Cloud recognizes three different [custom build scripts][] that can be
+used to perform additional tasks at a designated time. It also includes a set
+of [predefined environment variables][], such as `$CI_WORKSPACE`, which is the
+location of your cloned repository.
+
+{{site.alert.note}}
+  The temporary build environment that Xcode Cloud uses includes tools that are
+  part of macOS and Xcode&mdash;for example, Python&mdash;and additionally Homebrew to
+  support installing third-party dependencies and tools.
+{{site.alert.end}}
+
+#### Post-clone script
+
+Leverage the `post-clone` script that runs after Xcode Cloud
```

Comment:
I don't think `post-clone` should be wrapped in ticks since it's not code or a file name, it's just running after clone.
```suggestion
Leverage the post-clone script that runs after Xcode Cloud
```